config.js

```diff
@@ -6,38 +6,38 @@ var config = {
 
```

Comment:
Die Config wird durch Ansible geschrieben in soferen muss sie hier nicht ins git